test-data/unit/check-isinstance.test

```diff
@@ -1412,3 +1412,171 @@ def f(x: Union[int, A], a: Type[A]) -> None:
 
 [builtins fixtures/isinstancelist.pyi]
 
+
+[case testIssubclass]
+from typing import Type, ClassVar
+
+class Goblin:
+    aggressive: bool = True
+    level: int
```

Comment:
Do you need both attributes here? Some test cases could be simplified if you have only one instance variable.
Also you probably don't need to check both access and assignment to instance variables in all tests, just checking assignment is enough.